Sources/Quick/DSL/World+DSL.swift

```diff
@@ -50,42 +55,50 @@ extension World {
         currentExampleGroup.hooks.appendBefore(closure)
     }
 
+#if !os(Linux)
     @objc(beforeEachWithMetadata:)
     internal func beforeEach(closure closure: BeforeExampleWithMetadataClosure) {
         currentExampleGroup.hooks.appendBefore(closure)
     }
+#else
+    internal func beforeEach(closure closure: BeforeExampleWithMetadataClosure) {
+        currentExampleGroup.hooks.appendBefore(closure)
+    }
+#endif
 
     internal func afterEach(closure: AfterExampleClosure) {
         currentExampleGroup.hooks.appendAfter(closure)
     }
 
+#if !os(Linux)
     @objc(afterEachWithMetadata:)
     internal func afterEach(closure closure: AfterExampleWithMetadataClosure) {
         currentExampleGroup.hooks.appendAfter(closure)
     }
+#else
+    internal func afterEach(closure closure: AfterExampleWithMetadataClosure) {
+        currentExampleGroup.hooks.appendAfter(closure)
+    }
+#endif
 
-    @objc(itWithDescription:flags:file:line:closure:)
```

Comment:
Why remove these? I believe these were originally added to make the methods more readable in Objective-C:

```objc
// With the @objc name:
[[World sharedWorld] itWithDescription:@"foo" flags:@{} /* ... */];

// Without the @objc name:
[[World sharedWorld] itDescription:@"foo" flags:@{} /* ... */];
```